roles/ceph-osd/templates/ceph-osd.service.j2

```diff
@@ -5,8 +5,10 @@ After=docker.service
 [Service]
 EnvironmentFile=-/etc/environment
 ExecStartPre=-/usr/bin/docker stop {{ ansible_hostname }}-osd-dev%i
-ExecStartPre=-/usr/bin/docker rm -f  {{ ansible_hostname }}-osd-dev%i
-ExecStart=/usr/bin/docker run --rm --net=host --pid=host\
+ExecStartPre=-/usr/bin/docker rm -f {{ ansible_hostname }}-osd-dev%i
+ExecStart=/usr/bin/docker run --rm --net=host \
+   --cap-add SYS_ADMIN \
```

Comment:
@jimcurtis https://docs.docker.com/engine/reference/run/#/runtime-privilege-and-linux-capabilities

It's a bit vague as this is described as "allow admin operations" but still less privileges than using privileged mode. I need this to mount osd dir (see commit message).